src/PerfView/HeapView/GcInfoView.cs

```diff
@@ -129,6 +129,7 @@ public double PauseDuration
             }
         }
 
+        [Obsolete("This is experimental, you should not use it yet for non-experimental purposes.")]
```

Comment:
Adding @jeffschwMSFT how owns TraceGC.  

Jeff: we made TraceGC experimental by marking most of its methods Obsolete with the special message.   
Do we feel comfortable making most of it at least non-experimental?   We should discuss if there is overlap/conflict in the current APIs.